tests/cases/unittests/commandLineParsing.ts

```diff
@@ -0,0 +1,160 @@
+﻿/// <reference path="..\..\..\src\harness\harness.ts" />
+/// <reference path="..\..\..\src\compiler\commandLineParser.ts" />
+
+namespace ts {
+    describe('parseCommandLine', () => {
+
+        function assertParseResult(commandLine: string[], expectedParsedCommandLine: ts.ParsedCommandLine) {
+            const parsed = ts.parseCommandLine(commandLine);
+            const parsedCompilerOptions = JSON.stringify(parsed.options);
+            const expectedCompilerOptions = JSON.stringify(expectedParsedCommandLine.options);
+            assert.equal(parsedCompilerOptions, expectedCompilerOptions);
+
+            const parsedErrors = parsed.errors;
+            const expectedErrors = expectedParsedCommandLine.errors;
+            assert.isTrue(parsedErrors.length === expectedErrors.length, `Expected error: ${JSON.stringify(expectedErrors)}. Actual error: ${JSON.stringify(parsedErrors)}.`);
+            for (let i = 0; i < parsedErrors.length; ++i) {
+                const parsedError = parsedErrors[i];
+                const expectedError = expectedErrors[i]; 
+                assert.equal(parsedError.code, expectedError.code, `Expected error-code: ${JSON.stringify(expectedError.code)}. Actual error-code: ${JSON.stringify(parsedError.code)}.`);
+                assert.equal(parsedError.category, expectedError.category, `Expected error-category: ${JSON.stringify(expectedError.category)}. Actual error-category: ${JSON.stringify(parsedError.category)}.`);
+            }
+
+            const parsedFileNames = parsed.fileNames;
+            const expectedFileNames = expectedParsedCommandLine.fileNames;
+            assert.isTrue(parsedFileNames.length === expectedFileNames.length, `Expected fileNames: [${JSON.stringify(expectedFileNames)}]. Actual fileNames: [${JSON.stringify(parsedFileNames)}].`);
+            for (let i = 0; i < parsedFileNames.length; ++i) {

```

Comment:
As a heads up, you'll need to apply the linter to both unit test files. :smiley: